subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/tasks/CreateDependenciesFile.java

```diff
@@ -0,0 +1,61 @@
+package org.shipkit.internal.gradle.java.tasks;
+
+import java.util.Arrays;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.ModuleVersionIdentifier;
+import org.gradle.api.artifacts.ResolvedArtifact;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.TaskAction;
+import org.shipkit.internal.gradle.util.StringUtil;
+import org.shipkit.internal.notes.util.IOUtil;
+
+import java.io.File;
+
+public class CreateDependenciesFile extends DefaultTask {
+
+    @Input
```

Comment:
"InputFiles" annotation is the way of declaring FileCollection / Configuration input in Gradle. "Input" will not work correctly (unless Gradle folks have changed things :).

Reply:
Thanks, I didn't know that Configuration is a FileCollection!